AnimateSvg.js

```diff
@@ -14,6 +14,8 @@ export default function AnimateSvg({ isShakeTriggered, setIsShakeReady, hasIniti
   });
   const sizeAnim = useRef(new Animated.Value(1)).current;
 
+  const leftPx = width * leftPositionPercentDec;
```

Comment:
Dynamic calculation of component's `left` positioning.

Reply:
```suggestion
  const leftPosition = width * leftPositionPercentDec;
```

Also, why don't we need a `topPosition` for this component like the other one?